frontend/app/controllers/IdentityRequest.scala

```diff
@@ -7,16 +7,16 @@ case class IdentityRequest(headers: List[(String, String)], trackingParameters:
 
 object IdentityRequest extends RemoteAddress {
   def apply(request: Request[_]): IdentityRequest = {
-    val cookie = request.cookies.get("SC_GU_U")
     val ipAddress = clientIp(request)
 
-    val headers = List(("X-GU-ID-Client-Access-Token" -> s"Bearer ${Config.idApiClientToken}")) ++
-      cookie.map("X-GU-ID-FOWARDED-SC-GU-U" -> _.value)
+    val headers = List("X-GU-ID-Client-Access-Token" -> s"Bearer ${Config.idApiClientToken}") ++
+      request.cookies.get("SC_GU_U").map(_.value).map("X-GU-ID-FOWARDED-SC-GU-U" -> _) ++
+      request.headers.get("GU-IdentityToken").map("Authorization" -> _)
```

Comment:
So there will be a change in Identity to make the client-specific `GU-IdentityToken` allowed to perform the required actions? Add a link here when you create the PR, I'd like to see it!